doc/liblouis.texi

```diff
@@ -674,13 +679,28 @@ begcapsword 6-6
 @opcode{endcapsword, dots}
 The dot pattern which ends a block of capital letters within a word.
 It is used in cases where the block is not terminated automatically
-by a word boundary, a number or punctuation.
+by a word boundary, a number or punctuation, usually when a small
+letter appears later in the same word.
```

Comment:
This addition confused me a bit at first. I get it now but wonder if it adds much.